extensions/pipe-commands/src/manage-pipe-commands.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from "react";
+import { parseScriptCommands } from "./utils";
+import { List, Icon, ActionPanel, Action } from "@raycast/api";
+import { basename } from "path";
+import { ScriptCommand } from "./types";
+import { PipeCommand } from "./pipe-to-command";
+
+export default function managePipeCommands() {
+  const [state, setState] = useState<{ invalid: string[]; commands: ScriptCommand[] }>();
+
+  const loadCommands = () => {
+    parseScriptCommands().then(setState);
+  };
+
+  useEffect(loadCommands, []);
+  const textCommands = state?.commands.filter((command) => command.metadatas.input.type === "text");
+  const fileCommands = state?.commands.filter((command) => command.metadatas.input.type === "file");
+
+  return (
+    <List isLoading={typeof state == "undefined"}>
```

Comment:
In regards to the order of the sections, I'd use this order: "invalid", "file", "text". I'd display "invalid" at the top so the user could notice it and act to solve the issue with these commands.  And I'd put the "file" section before the "text" one cause I think these should be alphabetic ordered.